src/staff-administrator/components/staff-user-details/staff-user-details.component.ts

```diff
@@ -1,5 +1,6 @@
 import { Component } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
+import { Router } from '@angular/router';
```

Comment:
Two separate imports can be put in same line

Reply:
Updated.